packages/components/src/Layout/Semantics/semanticStyledBase.ts

```diff
@@ -34,7 +34,15 @@ import { css } from 'styled-components'
 
 export interface SemanticLayoutBase
   extends PaddingProps,
-    CompatibleHTMLProps<HTMLElement> {}
+    CompatibleHTMLProps<HTMLElement> {
+  /**
+   * scroll patterns for different areas inside the layout:
+   *  independently (default) | all together
+   * this prop has to be used in conjunction with the prop fixed.
+   * @default false
+   */
```

Comment:
Oh that's a good callout that it's meant to be used within a `<Page fixed>`! I think that could be a little more explicit about _which_ fixed prop you're referring to since that's a different component entirely. Maybe this?

```suggestion
  /**
   * To be used within the context of <Page fixed> container. 
   * When true, this removes the inner overflow-y scrolling
   * and allows content within a Layout group to scroll together.
   * @default false
   */
```

Reply:
thanks, naming things and explaining  use cases are so hard some times.